lib/rubocop/cop/root_cops/use_change.rb

```diff
@@ -0,0 +1,15 @@
+module RuboCop
+  module Cop
+    module RootCops
+      class UseChange < Cop
+        MSG = "Migration must have #change method.".freeze
+
+        def on_class(node)
+          return if node.each_descendant(:def).any? { |n| n.children[0] == :change }
```

Comment:
How does this ensure we're only checking classes that inherit from ActiveRecord::Migration? Would something like this fail?
```ruby
class Foo; end
```

Reply:
This cop is scope to `db/migrate/*.rb` so all files within this directory should inherit from `ActiveRecord::Migration`.

Reply:
root-server doesn't do migrations like that 🙁 might be easier to not scope and check that we're in a migration class. since we use this gem in other projects that do migrations in a normal way
```
✘-PIPE ~/code/root-monorepo/server [release-server|✔]
11:32 $ find . -iname '*migrate*'
./bin/migrate_file
./db/daveability/migrate
./db/skip_drive/migrate
./db/hard_brakes/migrate
./db/driver_passenger/migrate
./db/driveability/migrate
./db/telematics/migrate
./db/rating_plans/migrate
./db/telematics_segments/migrate
./db/distracted_driving/migrate
./db/location_changes/migrate
./db/drive_science/migrate
./db/rerating/migrate
./db/trip_summaries/migrate
./db/renters/migrate
./db/homeowners/migrate
./db/domain/migrate
./db/trip_tracker/migrate
```

Reply:
Fixed in https://github.com/Root-App/root-ruby-style/pull/31/commits/fee8131faa1fa122e7dac377ba3ff0c2878f392f